config/core/oracle.json

```diff
@@ -545,81 +417,8 @@
         "maxSubscriptionsPerConnection": 0
       },
       "type": "price_provider"
-    },
-    {
-      "name": "uniswapv3_api-ethereum",
-      "api": {
-        "enabled": true,
-        "timeout": 1000000000,
-        "interval": 2000000000,
-        "reconnectTimeout": 2000000000,
-        "maxQueries": 1,
-        "atomic": true,
-        "url": "https://eth.public-rpc.com/",
-        "endpoints": null,
-        "batchSize": 0,
-        "name": "uniswapv3_api-ethereum"
-      },
-      "webSocket": {
-        "enabled": false,
-        "maxBufferSize": 0,
-        "reconnectionTimeout": 0,
-        "wss": "",
-        "name": "",
-        "readBufferSize": 0,
-        "writeBufferSize": 0,
-        "handshakeTimeout": 0,
-        "enableCompression": false,
-        "readTimeout": 0,
-        "writeTimeout": 0,
-        "pingInterval": 0,
-        "maxReadErrorCount": 0,
-        "maxSubscriptionsPerConnection": 0
-      },
-      "type": "price_provider"
-    },
-    {
-      "name": "raydium_api",
-      "api": {
-        "enabled": true,
-        "timeout": 500000000,
-        "interval": 500000000,
-        "reconnectTimeout": 2000000000,
-        "maxQueries": 10,
-        "atomic": false,
-        "url": "",
-        "endpoints": [
-          {
-            "url": "https://api.devnet.solana.com",
-            "authentication": {
-              "apiKey": "",
-              "apiKeyHeader": ""
-            }
-          }
-        ],
-        "batchSize": 50,
-        "name": "raydium_api"
-      },
-      "webSocket": {
-        "enabled": false,
-        "maxBufferSize": 0,
-        "reconnectionTimeout": 0,
-        "wss": "",
-        "name": "",
-        "readBufferSize": 0,
-        "writeBufferSize": 0,
-        "handshakeTimeout": 0,
-        "enableCompression": false,
-        "readTimeout": 0,
-        "writeTimeout": 0,
-        "pingInterval": 0,
-        "maxReadErrorCount": 0,
-        "maxSubscriptionsPerConnection": 0
-      },
-      "type": "price_provider"
     }
   ],
-  "production": true,
```

Comment:
Is this field still used for default log level or did we override all of that? Can't remember what the `production` field actually controls at this point.

Reply:
its deprecated and not used anywhere